library/src/main/java/com/mapzen/android/OverlayManager.java

```diff
@@ -35,11 +35,13 @@
   private static final String NAME_POLYGON = "mz_default_polygon";
   private static final String NAME_MARKER = "mz_default_point";
   private static final String NAME_START_PIN = "mz_route_start";
-  private static final String NAME_END_PIN = "mz_route_stop";
+  private static final String NAME_END_PIN = "mz_route_destination";
```

Comment:
On a related note, we will need to update cinnabar and refill too. I'll do that and then push